src/backend/core/api/viewsets.py

```diff
@@ -293,3 +399,44 @@ class ResourceAccessViewSet(
     permission_classes = [permissions.ResourceAccessPermission]
     queryset = models.ResourceAccess.objects.all()
     serializer_class = serializers.ResourceAccessSerializer
+
+
+class RecordingViewSet(viewsets.GenericViewSet, mixins.ListModelMixin):
+    """
+    API endpoints to access and perform actions on recording.
+    """
+
+    storage = StorageHandler(
+        bucket_name=settings.AWS_STORAGE_BUCKET_NAME, parser_class=MinioParser
+    )
```

Comment:
Might be silly from me: The `StorageHandler` seems highly related to the worker, so maybe it should be deduced from the worker class? Or we expect all workers to use the same storage? For a first implementation, we can keep it as it.

Reply:
Actually it's not related to the worker!

Reply:
as of today, I expect all workers to use the same storage